### ISSUE TITLE
new_interframe compile issues with gcc/g++ windows

### ISSUE THREAD

**Issue by DoctorWho11** [EmperorArthur/VBA-M#14]
Here is a current build log of the latest changes you have made (with some of my own which is already been committed and a pull request issued) I'm working on a fix but figured I'd let you at least know.
`E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:17:10: error: expected unqualified-id before numeric constant
     u32 *frm1;
          ^
In file included from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:30:0,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:17:9: error: expected ';' at end of member declaration
     u32 *frm1;
         ^
In file included from E:/msys64/mingw32/i686-w64-mingw32/include/windows.h:84:0,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/wrapwin.h:51,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/app.h:92,
                 from E:/msys64/mingw32/include/wx-2.8/wx/app.h:562,
                 from E:/msys64/mingw32/include/wx-2.8/wx/wx.h:26,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxhead.h:14,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:9,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:17:10: error: expected unqualified-id before numeric constant
     u32 *frm1;
          ^
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:18:10: error: expected unqualified-id before numeric constant
     u32 *frm2;
          ^
In file included from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:30:0,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:18:9: error: expected ';' at end of member declaration
     u32 *frm2;
         ^
In file included from E:/msys64/mingw32/i686-w64-mingw32/include/windows.h:84:0,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/wrapwin.h:51,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/app.h:92,
                 from E:/msys64/mingw32/include/wx-2.8/wx/app.h:562,
                 from E:/msys64/mingw32/include/wx-2.8/wx/wx.h:26,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxhead.h:14,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:9,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:18:10: error: expected unqualified-id before numeric constant
     u32 *frm2;
          ^
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:19:10: error: expected unqualified-id before numeric constant
     u32 *frm3;
          ^
In file included from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:30:0,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:19:9: error: expected ';' at end of member declaration
     u32 *frm3;
         ^
In file included from E:/msys64/mingw32/i686-w64-mingw32/include/windows.h:84:0,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/wrapwin.h:51,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/app.h:92,
                 from E:/msys64/mingw32/include/wx-2.8/wx/app.h:562,
                 from E:/msys64/mingw32/include/wx-2.8/wx/wx.h:26,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxhead.h:14,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:9,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:19:10: error: expected unqualified-id before numeric constant
     u32 *frm3;
          ^
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:32:10: error: expected unqualified-id before numeric constant
     u32 *frm1;
          ^
In file included from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:30:0,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:32:9: error: expected ';' at end of member declaration
     u32 *frm1;
         ^
In file included from E:/msys64/mingw32/i686-w64-mingw32/include/windows.h:84:0,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/wrapwin.h:51,
                 from E:/msys64/mingw32/include/wx-2.8/wx/msw/app.h:92,
                 from E:/msys64/mingw32/include/wx-2.8/wx/app.h:562,
                 from E:/msys64/mingw32/include/wx-2.8/wx/wx.h:26,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxhead.h:14,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.h:9,
                 from E:/msys64/home/Zach/code/vba-m/src/wx/wxvbam.cpp:7:
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:32:10: error: expected unqualified-id before numeric constant
     u32 *frm1;
          ^
src/wx/CMakeFiles/wxvbam.dir/build.make:80: recipe for target 'src/wx/CMakeFiles/wxvbam.dir/wxvbam.cpp.obj' failed
make[2]: *** [src/wx/CMakeFiles/wxvbam.dir/wxvbam.cpp.obj] Error 1
CMakeFiles/Makefile2:162: recipe for target 'src/wx/CMakeFiles/wxvbam.dir/all' failed
make[1]: *** [src/wx/CMakeFiles/wxvbam.dir/all] Error 2
Makefile:116: recipe for target 'all' failed
make: *** [all] Error 2`

**Comment by EmperorArthur** [comment]
E:/msys64/home/Zach/code/vba-m/src/wx/../filters/new_interframe.hpp:17:10: error: expected unqualified-id before numeric constant u32 *frm1;

new_interframe.hpp #includes "../common/Types.h" which includes these two relevant lines:
#include <stdint.h>
typedef uint32_t u32;

**Comment by DoctorWho11** [comment]
Yeah... I know it's there, and it's included in new_interframe.hpp. Problem is still problem regardless.

**Comment by EmperorArthur** [comment]
Sorry I came off as kind of stiff there.
I hate working on these compiler/build environment issues.  
Hopefully, I've just fixed the issue, but if not I may have to set up a whole virtual machine just to figure out what's going on.

The big thing is that I see the filter framework as super important, especially with regards to the ease of adding new filters.

**Comment by DoctorWho11** [comment]
You know what I totally understand that. It's why I'm reporting these issues so that it can be fixed and properly used and when the time is right, merged right in. The VBA-M Team and I have come to a conclusion of using the wxwidgets interface by default when things start shaping up. I've been merging what I can. The biggest thing Squall wants to keep is the kega fusion plugins. So I've been merging your work into a branch eventually once I get a good grasp at what you're doing (which I must add, I'm starting to finally and I do like the idea) I'll need to re-add the kega plugins. The 16bit filters though, I don't think is a major loss. But because of your work, you've helped me spring into action as well with another developer who joined the team. Gonna put vba-m back on the map.
